src/p2f/src/CodeOwls.PowerShell/CodeOwls.PowerShell.Provider/Provider.cs

```diff
@@ -119,7 +125,8 @@ protected string EnsurePathIsRooted(string path)
             if (null != PSDriveInfo &&
                 !String.IsNullOrEmpty(PSDriveInfo.Root))
             {
-                var separator = PSDriveInfo.Root.EndsWith("\\") ? String.Empty : "\\";
+                var separator = PSDriveInfo.Root.EndsWith(StringSeparator) ? String.Empty : StringSeparator;
+
                 if (!path.StartsWith(PSDriveInfo.Root))
                 {
                     path = PSDriveInfo.Root + separator + path;
```

Comment:
Path.Combine will automatically insert the correct separator. Please use it instead. An added benefit is that line 128 can be removed as well.